### DISCUSSION TITLE
[Nhận task]

### DISCUSSION THREAD

**Discussion by nguyentantrung2000** [Runway-Club/Runway-CV-Builder#56] [Task Delivery]
Tôi và @trungdoublelift  và @nguyenxuanquang6789  sẽ nhận task Vẽ template cv1,cv2,cv3 để phục vụ cho việc tạo ra CV cho user

**Comment by nguyentantrung2000** [comment]
Thông tin task:
Phục vụ cho việc tạo ra CV cho user
Yêu cầu:
- Phải có hình mẫu của CV trước
- CV tạo ra phải giống hình mẫu của CV.
- Deadline 11:59PM  7/8/2021

**Reply by nguyentantrung2000** [reply]
OK

**Reply by trungdoublelift** [reply]
OK

**Reply by nguyenxuanquang6789** [reply]
Ok
